lib/android.js

```diff
@@ -908,7 +910,7 @@ function _getArtMethodSpec (vm) {
     const entrypointFieldSize = (apiLevel <= 21) ? 8 : pointerSize;
 
     const expectedAccessFlags = kAccPublic | kAccStatic | kAccFinal | kAccNative;
-    const allFlagsExceptPublicApi = ~kAccPublicApi >>> 0;
+    const allFlagsExceptPublicApi = ~(kAccPublicApi | kAccNterpInvokeFastPathFlag) >>> 0;
```

Comment:
Should rename this to match the new behavior. Perhaps something like `relevantAccessFlagsMask`.